docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -140,20 +140,62 @@ Regardless of the way you submit your work, follow these steps.
 .. _ticket tracker: https://code.djangoproject.com/
 .. _Development dashboard: https://dashboard.djangoproject.com/
 
-Non-trivial contributions
-=========================
+Contributions which require community feedback
+==============================================
 
-A "non-trivial" contribution is one that is more than a small bug fix. It's a
-change that introduces new Django functionality and makes some sort of design
-decision.
+A patch that introduces Django functionality and makes some sort of design
+decision requires a wider community discussion. Especially if the approach
+involves a deprecation or potentially introducing breaking changes.
```

Comment:
I'm wondering if we should turn this into a bullet list and add the "needs strong consensus" examples. 
Another example beyond depredations and breaking changes would be "adding a new setting" (as many people suggest to "just" add a setting and we try to avoid having too many settings in Django)

Reply:
Yes, Will prefer that too. It would be easier to read if we have it as a list

Reply:
I have changed it to a list, and it looks better and more readable. Thank you. 

I'm not  sure I understand the examples you mentioned, could you help with how it should look like?